pywikibot/site.py

```diff
@@ -4400,11 +4400,15 @@ def recentchanges(self, start=None, end=None, reverse=False,
     @deprecated_args(number='total', key='searchstring',
                      getredirects='get_redirects')
     def search(self, searchstring, namespaces=None, where="text",
-               get_redirects=False, step=None, total=None, content=False):
+               get_redirects=False, step=None, total=None, content=False, 
+               checkSnippet=False, skipAfterNrMisses=-1):
```

Comment:
we are now using pep8 compliant names, so it should be `check_snippet` and `skip_after_misses`(?). 
The default for `skip_after_misses` should be negatives like False, None or 0, rather than -1.